docs/resources/resources-github-copilot-prompt-library.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Office Add-ins prompts for GitHub Copilot
+description: An Office Add-ins prompt library to use with GitHub Copilot.
+ms.date: 05/16/2024
+ms.localizationpriority: medium
+---
+
+# Office Add-ins prompts for GitHub Copilot
+
+The Office Add-ins prompt library is a collection of prompt examples to use with [GitHub Copilot](https://github.com/features/copilot/plans) when you develop Office add-ins. GitHub Copilot is an AI pair-programmer that helps you write code. It's available as an extension for many IDEs, such as [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) and [Visual Studio](https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs). You can install and launch the extensions directly through your IDE.
+
+Copy the prompts from this article and enter them into the GitHub Copilot chat to begin experimenting. You can also open the prompt list in your IDE by downloading the file: [link to GitHub repo].
```

Comment:
The link should be replaced with the GH page of this document after publishment, right?

Reply:
Yes, added it to the article